app/themes/user-admin/src/app/users-event-log/users-event-log.component.html

```diff
@@ -0,0 +1,62 @@
+<div class="row">
+    <div class="col-xs-4">
+        <button class="btn btn-default btn-sm" routerLink="/user/">Create a New User</button>
+    </div>
+</div>
+<div class="row">
+    <div class="col-xs-12">
+        <br />
+    </div>
+</div>
+<div class="row">
+    <div class="col-xs-12">
+        <table class="table table-bordered table-hover"> 
+            <thead>
+                <tr>
+                    <th (click)="sort()">
+                        Created At
+                        <i class="fa" [ngClass]="{'fa-sort-asc': (sortDir == 'asc'), 'fa-sort-desc': (sortDir == 'desc')}" aria-hidden="true"></i>
+                    </th>
+                    <th>
+                        User ID
+                    </th>
+                    <th>
+                        ID
+                    </th>
+                    <th>
+                        Event Log
+                    </th>
+                    <th>
+                        Event Data
+                    </th>
+                </tr>
+            </thead>
+
+            <tbody *ngIf="wait"> 
+                <tr>
+                    <td colspan="7" align="center">
+                        <i class="fa fa-refresh fa-spin fa-2x fa-fw"></i> 
+                    </td>
+                </tr>
+            </tbody>
+
+            <tbody *ngIf="!wait && events.length <= 0">
+                <tr>
+                    <td colspan="7" align="center">
+                        <alert type="warning">Sorry, no user events were found.</alert>
+                    </td>
+                </tr>
+            </tbody>
+
+            <tbody *ngIf="!wait && events.length > 0">
+                <tr class="users-events" *ngFor="let event of events">
+                    <td> {{event.created_at}} </td>
+                    <td> {{event.user_id}} </td>
+                    <td> {{event.id}} </td>
+                    <td> {{event.event}} </td>
+                    <td> {{event.event_data|json}} </td>
+                </tr>
+            </tbody>
+        </table>
```

Comment:
We need to add pagination to this element to handle multiple pages of logs.

I do have to apologize that the AJAX response was not actually returning the total counts for the log entries. I have pushed an update to the `users` repo including this in the response. Let me know if you have trouble seeing these updates when you run `composer update`